code/modules/mob/living/brain/brain_item.dm

```diff
@@ -181,32 +181,37 @@
 
 	add_fingerprint(user)
 
-	if(user.zone_selected != BODY_ZONE_HEAD)
+	if(user.zone_selected != zone)
 		return ..()
 
 	var/target_has_brain = C.getorgan(/obj/item/organ/brain)
 
-	if(!target_has_brain && C.is_eyes_covered())
+
+	// This should be a better check but this covers 99.9% of cases
+	if((C?.dna?.species?.inherent_biotypes & MOB_ROBOTIC && status != ORGAN_ROBOTIC) || (C?.dna?.species?.inherent_biotypes & MOB_ORGANIC && status != ORGAN_ORGANIC))
```

Comment:
use the new process flags variables

Reply:
...why?

Reply:
Because it's a much simpler check that catches more edge cases: `!(process_flags & C.get_process_flags())`

The way you've done it will break if you try to put any kind of brain into a preternis since they have mob_robotic biotype but have an organic brain.

Reply:
I hate mob code so much :^

Reply:
Atleast its simpler to use

Be nice to have a organ type flag that organs and species use